src/ontology/cl-edit.owl

```diff
@@ -32672,6 +32673,16 @@ SubClassOf(obo:CL_4052010 obo:CL_0000499)
 SubClassOf(obo:CL_4052010 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0012186))
 SubClassOf(obo:CL_4052010 ObjectSomeValuesFrom(obo:RO_0002203 obo:CL_0000503))
 
+# Class: obo:CL_4052011 (follicle associated enterocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29339448") Annotation(oboInOwl:hasDbXref "PMID:7107724") Annotation(oboInOwl:hasDbXref "doi:/10.1101/2021.04.06.438564") obo:IAO_0000115 obo:CL_4052011 "An enterocyte found in the follicle-associated epithelium (FAE) that covers Peyer's patches and other mucosa-associated lymphoid tissues. This cell has reduced absorptive capacity, a thinner glycocalyx layer, and expresses higher levels of chemokines CCL20 and CXCL16, compared to regular villus enterocytes. It contributes to antigen sampling and immune interactions, supporting the specialized function of the FAE in mucosal immunity.")
```

Comment:
I could not find the following statements in the definition in any of the three publications:
> This cell has reduced absorptive capacity, a thinner glycocalyx layer

(I could find reduce absorptive nutrient genes, but nothing about glycocalyx).